dbschema/comment-thread.esdl

```diff
@@ -0,0 +1,29 @@
+module default {
+  type Comment extending Resource, Mixin::Owned {
+    required body: default::RichText;
+    required thread: CommentThread;
+  }
+
+  type CommentThread extending Resource, Mixin::Embedded, Mixin::Owned {
```

Comment:
Following suit of other things, like BudgetRecord, this could be `Comment::Thread`.
aka `module Comment { type Thread }`. It does feel a little weird since Thread is the entry point. But the name is nice. Thoughts?

Reply:
@CarsonF what about `Comment`? Would that become `Comment::Comment`?

Reply:
I guess that stays `default::Comment`

Reply:
Andrew and I had a discussion about this last Thursday on the first few passes of this schema. Where we are now is if it makes sense to switch up the module and naming convention for how this is implemented then we're fine with the change. It is a matter of opinion and either way makes sense to me

Reply:
Feels like naming inconsistency, having `Thread` (which is the main of the comment) in the `Comment` namespace but `Comment` in the default.
There are just thoughts, I'm not leaning either way. Ok with both options.

Reply:
Yeah I agree Andrew. And Commentable is in a third module :/